includes/classes/SettingsApi.php

```diff
@@ -64,20 +85,22 @@ class SettingsApi {
 	/**
 	 * Constructor.
 	 *
-	 * @param string $page_title Page title for the settings page.
-	 * @param string $menu_title Menu title for the settings page.
-	 * @param string $capability Capability for the settings page.
-	 * @param string $slug Slug for the settings page.
-	 * @param int|null $position Menu position for the settings page.
+	 * @param string   $page_title  Page title for the settings page.
+	 * @param string   $menu_title  Menu title for the settings page.
+	 * @param string   $capability  Capability for the settings page.
+	 * @param string   $slug        Slug for the settings page.
+	 * @param int|null $position    Menu position for the settings page.
+	 * @param bool     $top_level   If it's a top level menu.
 	 */
-	public function __construct( $page_title, $menu_title, $capability, $slug, $position = null ) {
+	public function __construct( $page_title, $menu_title, $capability, $slug, $position = null, $top_level = false ) {
```

Comment:
Looks like a new `$icon_url` parameter was added but needs adding to the constructor too